ModularTegustation/tegu_quirks.dm

```diff
@@ -0,0 +1,80 @@
+//Prosthetic Limb
+//Right Arm
+/datum/quirk/prosthetic_limb/r_arm
+	name = "Prosthetic Right Arm"
+	desc = "An accident caused you to your right arm. Because of this, you now have a prosthetic arm!"
+	value = -4
+	medical_record_text = "During physical examination, patient was found to have a prosthetic right arm."
+	hardcore_value = 3
+
+/datum/quirk/prosthetic_limb/r_arm/on_spawn()
```

Comment:
This proc should be on /datum/quirk/prostethic_limb, so it triggers for every kind of limb and saves space.
Then you just add every possible limb in switch(limb_slot).

Reply:
Wouldn't that randomly select a limb with all 4 quirks?